### PR TITLE
increase wait time for tests

### PR DESCRIPTION
**Pull request by johrstrom**
These tests are flaky because they do not use the wait time at all. `assert_selector` is Minitest thing that doesn't respond to or use wait times. It executes immediately on the dom.

`find` on the other _does_ respond to wait time and `exact_text`, so we can use both to safely verify. `find` will throw an error if it cannot find the thing it's looking for so it's as good as an `assert`.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1202200629166196) by [Unito](https://www.unito.io)

### CODE REVIEW

File: apps/dashboard/test/system/files_test.rb [hunk 1]
```diff
@@ -65,19 +65,16 @@ class FilesTest < ApplicationSystemTestCase
...
+      find('tbody a', exact_text: 'app', wait: MAX_WAIT)
+      find('tbody a', exact_text: 'config', wait: MAX_WAIT)
+      find('tbody a', exact_text: 'manifest.yml', wait: MAX_WAIT)
+
+      # with copying done, let's assert on the UI and the file system
+      assert_selector 'span', text: '100% copy files', count: 1
       assert_equal "", `diff -rq #{File.join(dir, 'app')} #{Rails.root.join('app').to_s}`.strip, "failed to recursively copy app dir"
       assert_equal "", `diff -rq #{File.join(dir, 'config')} #{Rails.root.join('config').to_s}`.strip, "failed to recursively copy config dir"
       assert_equal "", `diff -q #{File.join(dir, 'manifest.yml')} #{Rails.root.join('manifest.yml').to_s}`.strip, "failed to copy manifest.yml"
-
```
Comment (gbsoftwaresolutions) [inline apps/dashboard/test/system/files_test.rb:78]: @johrstrom .  Did you not want to check to ensure the "100% copy files" disappears?

Comment (johrstrom) [reply apps/dashboard/test/system/files_test.rb:78]: I can try, but had to add a `sleep` here. I guess it's a balance between reliability (the test is always deterministic) and coverage.

Comment (gbsoftwaresolutions) [reply apps/dashboard/test/system/files_test.rb:78]: You didn't have to add it back. I was just wondering why?

File: apps/dashboard/test/system/files_test.rb [hunk 2]
```diff
@@ -65,19 +65,19 @@ class FilesTest < ApplicationSystemTestCase
...
+      # with copying done, let's assert on the UI and the file system
+      assert_selector 'span', text: '100% copy files', count: 1
       assert_equal "", `diff -rq #{File.join(dir, 'app')} #{Rails.root.join('app').to_s}`.strip, "failed to recursively copy app dir"
       assert_equal "", `diff -rq #{File.join(dir, 'config')} #{Rails.root.join('config').to_s}`.strip, "failed to recursively copy config dir"
       assert_equal "", `diff -q #{File.join(dir, 'manifest.yml')} #{Rails.root.join('manifest.yml').to_s}`.strip, "failed to copy manifest.yml"
-      
-      # we need to wait due to the nature of how the "% copy files" hides
-      assert_selector 'span', text: 'copy files', count: 0, wait: MAX_WAIT
+
+      sleep 6 # gott to gaurentee that this dissappears after 5 seconds.
```
Comment (gbsoftwaresolutions) [inline apps/dashboard/test/system/files_test.rb:79]:
```suggestion
      sleep 6 # need to guarantee that this disappears after 5 seconds.
```

File: apps/dashboard/test/system/files_test.rb (hunk 1 above)
Comment (johrstrom) [reply apps/dashboard/test/system/files_test.rb:78]: Coverage is good. I'm OK with the sleep - until it starts breaking

**Review by gbsoftwaresolutions — APPROVED** [review]
looks good. runs good. tests good.

**Comment by johrstrom** [comment]
Thanks, let's hope it keeps.

**Pull request merged**